src/components/vote_history_table/ElectionWithMemberVotesTable.tsx

```diff
@@ -0,0 +1,187 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import SimpleTable from '../SimpleTable'
+import { Styles } from 'src/libs/theme'
+import { formatDate, sortVisibleTable } from '../../libs/utils'
+import { ElectionWithMemberVotes, Vote } from 'src/types/model'
+import { ExpandMore, ExpandLess } from '@mui/icons-material'
+import VoteSummaryTable from '../vote_summary_table/VoteSummaryTable'
+
+interface ElectionWithMemberVotesTableProps {
+  electionsWithMemberVotes: ElectionWithMemberVotes[]
+}
+
+interface RowData {
+  data: string
+  cellStyle: React.CSSProperties
+  label: string
+  id: number | string
+  electionId?: number
+  memberVotes?: Vote[]
+  onClick?: () => void
+}
+
+const styles = {
+  baseStyle: {
+    fontFamily: 'Montserrat',
+    fontSize: '1.4rem',
+    fontWeight: 400,
+    color: '#333F52',
+    backgroundColor: '#FFFFFF',
+    display: 'flex',
+    padding: '1rem 2%',
+    lineHeight: '2rem',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    whiteSpace: 'pre-line',
+  },
+  columnStyle: {
+    ...Styles.TABLE.HEADER_ROW,
+    ...{
+      fontFamily: 'Montserrat',
+      fontSize: '1.4rem',
+      color: '#333F52',
+      justifyContent: 'space-between',
+    },
+  },
+  containerOverride: {
+    marginTop: '0',
+    borderTop: '0',
+    backgroundColor: 'rgba(184, 205, 211, 0)',
+    padding: '0',
+  },
+}
+
+const processVotesCast = (memberVotes: Vote[]) => {
+  if (!memberVotes || memberVotes.length === 0) {
+    return '0'
+  }
+  const total = memberVotes.length
+  const cast = memberVotes.filter(v => v.vote !== null && v.vote !== undefined).length
+  return `${cast}/${total}`
+}
+
+const processVoteSummary = (memberVotes: Vote[]) => {
+  if (!memberVotes || memberVotes.length === 0
+    || memberVotes.every(v => v.vote === null || v.vote === undefined)) {
+    return 'No votes cast'
+  }
+  const positives = memberVotes.filter(v => v.vote === true).length
+  const negatives = memberVotes.filter(v => v.vote === false).length
+  return positives + ' Approved, ' + negatives + ' Denied'
```

Comment:
Should we use the words 'YES' and 'NO' to match the words on the buttons the user clicks to vote?

<img width="465" height="126" alt="image" src="https://github.com/user-attachments/assets/a88d6b5e-4349-4e54-a1cd-9724a9242724" />

Reply:
Using Yes and No also matches the wording used in the tables on the member and chair tabs.